test/parse.js

```diff
@@ -1,206 +1,207 @@
 'use strict';
 
-var test = require('tap').test;
+var chai = require('chai');
+var expect = chai.expect;
+chai.should();
 
 var parse = require('../').parse;
 
-test('using connection string in client constructor', function(t){
-  var subject = parse('postgres://brian:pw@boom:381/lala');
-  t.equal(subject.user,'brian');
-  t.equal(subject.password, 'pw');
-  t.equal(subject.host, 'boom');
-  t.equal(subject.port, '381');
-  t.equal(subject.database, 'lala');
-  t.end();
+describe('parse', function(){
+
+  it('using connection string in client constructor', function(done){
```

Comment:
None of these tests are async so you can remove the done arg as well as the done call at the end.